src/test/java/net/mcreator/integration/generator/GTModElements.java

```diff
@@ -58,8 +58,10 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 			List<GeneratableElement> modElementExamples = TestWorkspaceDataProvider.getModElementExamplesFor(workspace,
 					modElementType, false, random);
 
-			LOG.info("[" + generatorName + "] Testing mod element type generation " + modElementType.getReadableName()
-					+ " with " + modElementExamples.size() + " variants");
+			if (LOG != null) {

```

Comment:
LOG here should never be null

Reply:
Is it still there? The file is no longer on the PR changes list on my device.

Reply:
No it is not, I did not have time to do another full round of checks yet